templates/gitlab.rb.j2

```diff
@@ -34,6 +34,17 @@ gitlab_rails['ldap_password'] = '{{ gitlab_ldap_password }}'
 gitlab_rails['ldap_allow_username_or_email_login'] = true
 gitlab_rails['ldap_base'] = '{{ gitlab_ldap_base }}'
 
+##################
+# GitLab Unicorn #
```

Comment:
For this section, can you just leave one comment line, like `# GitLab Unicorn settings` — that would better match the formatting of the rest of the file.